test/extended/builds/docker-build-no-outputname.go

```diff
@@ -0,0 +1,39 @@
+package builds
+
+import (
+	"fmt"
+	"path/filepath"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("builds: Docker image buildconfig without output.to.Name specified", func() {
+	defer g.GinkgoRecover()
+	var (
+		dockerImageFixture = filepath.Join("fixtures", "test-docker-no-outputname.json")
+		oc                 = exutil.NewCLI("build-docker-no-outputname", exutil.KubeConfigPath())
+	)
+
+	g.Describe("Building from a template", func() {
+		g.It(fmt.Sprintf("should create an image from %q template without an output image reference defined", dockerImageFixture), func() {
+			oc.SetOutputDir(exutil.TestContext.OutputDir)
+			err := oc.Run("create").Args("-f", dockerImageFixture).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("starting a build")
+			buildName, err := oc.Run("start-build").Args("test").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("expecting the build is in Complete phase")
+			err = exutil.WaitForABuild(oc.REST().Builds(oc.Namespace()), buildName, exutil.CheckBuildSuccessFunc, exutil.CheckBuildFailedFunc)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("expecting build to pass without an output image reference specified")
+			out, err := oc.Run("start-build").Args("test", "--follow").Output()
```

Comment:
i thought about that, but didn't know what to chose between adding some additional code to a rather simple test or just get what i need from the current one, however i will refactor accordingly.